usaspending_api/api_docs/api_documentation/download/README.md

```diff
@@ -0,0 +1,40 @@
+# Download
+
+The following endpoints are involved with generating files that reflect the site's underlying data. 
+
+## Award Data Archive
+
+On a monthly basis, the website pre-generates a series of commonly used files based on the agency, fiscal year, and award type. These can be found at [Award Data Archive](https://beta.usaspending.gov/#/download_center/award_data_archive), or, if you prefer, you can also use the API's [List Downloads Endpoint](list_downloads.md).
+
+## Generating Download Files
+
+**Reminder**: Before using these endpoints, check the  [Award Data Archive](https://usaspending.gov/#/download_center/award_data_archive) for pre-generated files
```

Comment:
Missing punctuation at the end of this sentence

Reply:
done